index.html

```diff
@@ -0,0 +1,290 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title>Crypto Con</title>
+    <script
+      defer
+      src="https://use.fontawesome.com/releases/v5.3.1/js/all.js"
+    ></script>
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/bulma/0.9.4/css/bulma.min.css"
+    />
+    
+    <script defer type="text/javascript" src="/app.js"></script>
+    <!-- <link rel="stylesheet" href="debug.css" /> -->
+    
+    <link rel="stylesheet" href="styles.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <!-- <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />  -->
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato&display=swap"
+      rel="stylesheet"
+    />
+    <link href="https://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet" />
+  </head>
+  <body>
+    <!-- top banner -->
+    <section class="hero is-dark is-hidden-mobile pr-4">
+      <div class="columns">
+        <div class="column is-11">
+          <ul>
+            <li>
+              <figure class="image is-24x24">
+                <img src="images/facebook_f_64px.png" />
+              </figure>
+            </li>
+            <li>
+              <figure class="image is-24x24">
+                <img src="images/twitter_64px.png" />
+              </figure>
+            </li>
+            <li id="eng">English</li>
+            <li id="mypage">MyPage</li>
+            <li id="logout">Logout</li>
+          </ul>
+        </div>
+      </div>
+    </section>
+
+    <!-- navbar section -->
```

Comment:
- As the previous reviewer mentioned the contents of the page need to be well aligned as seen in the previous design but there are still some elements that aren't properly aligned so kindly fix them :+1: 
- Your Design
![image](https://github.com/Drew246/capstone-project-1/assets/102699958/99fda18d-e3bf-4e49-a1dc-0574b8ac7e1a)
 - [ ] In the second section I noticed that the previous reviewer mentioned that you style that section properly as seen in the Behance design but this wasn't implemented and also, that section is not properly in line with the rest the contents of the page I have pointed out some of the changes that need to be made below :+1: 
 
- Your Design
![image](https://github.com/Drew246/capstone-project-1/assets/102699958/62df5c3b-af10-45ed-b0b8-930c383ad49c)

Reply:
- [ ] As suggested by the previous reviewer that section is not properly in line with the rest the contents of the page Kindly refer the above pointed out some of the changes that need to be made above on the screenshot, Kindly Fix the issues.